src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,17 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts all persons in list.
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>(){
+            public int compare(Person p1, Person p2) {
+                return p1.getName().toString().compareTo(p2.getName().toString());
```

Comment:
Based on my understanding of how Java compares Strings, this would end up putting, for example, "AB" before "Aa". Is this intended, or should the sort be case-insensitive?